packages/core/src/utils.ts

```diff
@@ -258,6 +259,9 @@ export function resolveReferencedActor(machine: AnyStateMachine, src: string) {
   ).src;
 }
 
+/**
+ * @internal
+ */
```

Comment:
does it have any effect? We are actually exporting this as `__unsafe_getAllOwnEventDescriptors` - but I'm sure if this is applied transitively and even if it is then we should just... remove that export instead of marking this as internal here.